### PR TITLE
Release 0.6.4

### PR DESCRIPTION
**Pull request by paulcadman**
This PR updates:

- [x] Package version
- [x] Smoke test
- [x] Changelog

### CODE REVIEW

**Review by janmasrovira — APPROVED** [review]

**Comment by paulcadman** [comment]
We can't merge this until we merge:

* https://github.com/anoma/juvix-stdlib/pull/105

And I update the stdlib reference in this repo.

**Pull request merged**
